app/views/pages/admin_members.html.slim

```diff
@@ -15,4 +15,4 @@ main.admin-members
                 h3.title
                   = member.name
                 = link_to "@#{member.username}", "slack://user?team=#{ENV["SLACK_TEAM"]}&id=#{member.uid}", class: "has-text-primary"
-
+                p = member.pronouns
```

Comment:
Is it desirable to output an empty `<p>` tag here, in the case where the attribute is empty?

If not, something like this:

```
- if member.pronouns.present?
  p = member.pronouns
```

Reply:
Hmm, perhaps yes: for spacing?